webpack/assets/javascripts/bundle.js

```diff
@@ -6,6 +6,7 @@ require('jquery.cookie');
 require('./bundle_flot');
 require('./bundle_select2');
 require('./bundle_datatables');
-window.tfm = {
-  tools: require('./foreman_tools')
-};
+
+window.tfm = window.tfm || {};
```

Comment:
Why not just using an object with attributes like before? Is there any way that `window.tfm` may already be defined at this point?